src/client/helpers/entity.js

```diff
@@ -225,3 +226,59 @@ export function getISBNOfEdition(entity) {
 export function getEditionFormat(entity) {
 	return (entity.editionFormat && entity.editionFormat.label) || '?';
 }
+
+// relationshipTypeId = 10 refers the relation (<Work> is contained by <Edition>)
+const relationshipTypeId = 10;
+
+/**
+ * Filter the relationship according to relationshipTypeId
+ *
+ * @param {object} entity - Entity with all relationships
+ * @returns {object} retrun the all relationshiops after removing the relatioship with relationshipTypeId = 10
+ */
+export function getFilteredRelationship(entity) {
```

Comment:
I think it should be easy to make this function more reusable and pass a second `relationshipTypeId` argument instead of hardcoding it.

Reply:
If you're interested, I think it would be quite neat to use an enum that can be used when calling this function, as well as other places that rely on a relationshipTypeId number.

It is out of the scope of this PR, though, and I'll create a separate ticket for it.
If you're interested in diving a bit more into flow (the type system BookBrainz uses), you can have a look at https://alligator.io/react/flow-enums/#creating-an-enum-from-an-existing-value